doc/sphinx/documentation/tutorials/3DED/lysozyme_nanocrystals.rst

```diff
@@ -0,0 +1,822 @@
+#####################
+Lysozyme nanocrystals
+#####################
+
+.. highlight:: none
+
+Introduction
+============
+
+DIALS has been successfully adapted for processing electron diffraction data
+from protein nanocrystals. Extensions to the software and protocols for dealing
+with the peculiarities of electron diffraction data are described in a
+publication:
+
+.. pubmed:: 29872002 Electron diffraction
+
+.. warning::
+  This tutorial reproduces the data processing results described in that paper,
+  which were produced by DIALS version ``1.dev.2084-g06727c3`` and CCP4 version
+  ``7.0.051``. Results may differ with other versions of the software.
+
+The commands listed here assume the use of a Bash shell on a POSIX-compliant
+system, so would have to be adjusted appropriately for use on other systems
+such as Windows.
+
+Data
+====
+
+The data for this tutorial are available online at
+|lysozyme_nano|. There are seven datasets from different
+lysozyme nanocrystals stored as gzipped tar archives. These should be
+downloaded and expanded to create seven directories containing the images,
+``Lys_ED_Dataset_{1..7}``. We recommend doing data processing for each dataset
+in its own directory, separate from the images, for example
+``Lys_ED_Dataset_1-dials-proc``. When referring to the data directories in
+commands listed in this tutorial, we shall use an environment variable pointing
+to the parent directory. Assuming the current directory contains all of the
+downloaded and unpacked dataset directories, we set that variable as follows in
+a bash shell:
+
+.. |lysozyme_nano| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1250447.svg
+                  :target: https://doi.org/10.5281/zenodo.1250447
+
+.. code-block:: bash
+
+  export DATA_PARENT=$(pwd)
+
+Import
+======
+
+The images are in the standard miniCBF format, however the detector is unusual.
+We have constructed a special dxtbx ``Format`` class to interpret the images
+properly. This is not distributed with DIALS, but can be found in a separate
+repository here: https://github.com/dials/dxtbx_ED_formats.
+
+We can make use of the dxtbx runtime plug-in system to pick the format class
+up automatically by saving it in a special directory, ``$HOME/.dxtbx/`` (for
+POSIX-compliant systems). For example (may require installation of curl):
+
+.. code-block:: bash
+
+  if [ ! -f ~/.dxtbx/FormatCBFMiniTimepix.py ]; then
+    mkdir -p ~/.dxtbx
+    curl -o ~/.dxtbx/FormatCBFMiniTimepix.py https://raw.githubusercontent.com/dials/dxtbx_ED_formats/master/FormatCBFMiniTimepix.py
+  fi
```

Comment:
Don't we have to run `dxtbx.install_format`?

Reply:
Hi @biochem-fan. Yes. This particular tutorial is out of date. I put it online originally as a promise to reviewers to reproduce the processing commands used for the "DIALS for ED" paper, but have not updated it for modern DIALS installations. There is a warning at the top saying it was prepared with DIALS 1.dev.2084-g06727c3 and CCP4 version 7.0.051. I would like to process the data with modern DIALS and then have versions of the tutorials, like we do for the main ones like https://dials.github.io/documentation/tutorials/processing_in_detail_betalactamase.html. I just haven't had time to do that yet.

The main content of this PR is to introduce a new tutorial that *does* work with current DIALS.

Reply:
I should note, the reason `lysozyme_nanocrystals.rst` appeared as a new file is because I moved it from `dials_for_ed.rst`, but reused that filename as a new landing page for 3DED tutorials. The content of that file is not new though.